src/khoj/processor/tools/online_search.py

```diff
@@ -1,23 +1,25 @@
+import asyncio
 import json
 import logging
 import os
-from typing import Dict, List, Union
+from typing import Dict, Tuple, Union
 
+import aiohttp
 import requests
+from bs4 import BeautifulSoup
```

Comment:
Use beautifulsoup, rather than bs4

Reply:
Even though the pip package to install is `beautifulsoup4`, the package name for import is `bs4`. So this will stay the same